packages/flutter/test/material/bottom_navigation_bar_test.dart

```diff
@@ -2502,6 +3047,80 @@ void main() {
+    const double iconHeight = 20;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData(),
+        home: Builder(
+          builder: (BuildContext context) {
+            return Scaffold(
+              bottomNavigationBar: BottomNavigationBar(
+                landscapeLayout: BottomNavigationBarLandscapeLayout.linear,
+                items: <BottomNavigationBarItem>[
+                  BottomNavigationBarItem(
+                    icon: SizedBox(key: icon0, width: iconWidth, height: iconHeight),
+                    label: 'Title0',
+                  ),
+                  BottomNavigationBarItem(
+                    icon: SizedBox(key: icon1, width: iconWidth, height: iconHeight),
+                    label: 'Title1',
+                  ),
+                ],
+              ),
+            );
+          },
+        ),
+      ),
+    );
+
+    expect(tester.getSize(find.byType(BottomNavigationBar)), const Size(800, kBottomNavigationBarHeight));
+    expect(tester.getRect(find.byType(BottomNavigationBar)), const Rect.fromLTRB(0, 600 - kBottomNavigationBarHeight, 800, 600));
+
+    const double navigationBarTop = 600 - kBottomNavigationBarHeight; // 544
+    const double selectedFontSize = 14.0;
+    const double m3LineHeight = 1.43;
+    final double labelHeight = (selectedFontSize * m3LineHeight).floorToDouble(); // 20
+    const double navigationTileVerticalPadding = selectedFontSize / 2; // 7.0
+    // Icon and label are in the same row.
+    final double navigationTileHeight = max(iconHeight, labelHeight) + 2 * navigationTileVerticalPadding;
+
+    // Navigation tiles parent is a Row with crossAxisAlignment sets to center.
+    final double navigationTileVerticalOffset = (kBottomNavigationBarHeight - navigationTileHeight) / 2;
+
+    final double iconTop = navigationBarTop + navigationTileVerticalOffset + navigationTileVerticalPadding;
+    final double labelBottom = 600 - (navigationTileVerticalOffset + navigationTileVerticalPadding);
+
+    // The items are laid out as in the default case except each
+    // item's icon/label is arranged in a row, with 8 pixels in
+    // between the icon and label.  The layout of the unselected
+    // item's label is slightly different; not checking that here.
+    const double itemFullWith = 800 / 2; // Two items in the navigation bar.
+    const double separatorWidth = 8;
+    final double firstLabelWidth = tester.getSize(find.text('Title0')).width;
+    final double firstItemContentWidth = iconWidth + separatorWidth + firstLabelWidth;
+    final double firstItemLeft = itemFullWith / 2 - firstItemContentWidth / 2;
+    final double secondLabelWidth = tester.getSize(find.text('Title1')).width;
+    final double secondItemContentWidth = iconWidth + separatorWidth + secondLabelWidth;
+    final double secondItemLeft = itemFullWith + itemFullWith / 2 - secondItemContentWidth / 2;    //const double firstLabelCenter = firstItemLeft +  itemWidth / 2; // 250
```

Comment:
```suggestion
    final double secondItemLeft = itemFullWith + itemFullWith / 2 - secondItemContentWidth / 2;
```

Reply:
Oh I was hoping I could commit this, but it appears the PR does not have permissions enabled for me to edit it. Pushing a new commit (not force pushing, and not rebasing) would fix the Gold issue here as well.

Reply:
Good catch for the extra comment! :+1: 

> (not force pushing, and not rebasing)
Thanks for your help.
I tried doing so but there were several tests failures (this PR was far from the tip of the tree, the failures were related to some changes on the CI configuration on Mac).

Let see how it goes after the rebase, I will try to use my new Gold tool knowledge :smile: and I will ask for your help if the PR is still stuck.